tema1/script.js

```diff
@@ -0,0 +1,26 @@
+console.log("EXERCITIUL 1");
+
+var variabila = "a";
+
+console.log("a" + " " + "a fost declarat");
+
+//console.log("valoarea lui a este ${a}");
+a = 10;
```

Comment:
Ca si informatie, problema de mai sus, duce la o problema aici si anume la faptul ca variabila ```a``` nu este declarata nicaieri(deoarece singura variabila declarata este ```variabila```).
In mod normal alte limbaje de programare nu ti-ar permite sa faci asta(specific, sa atribui o variabila care nu exista), dar JavaScript fiind compus din mai multe versiuni, iti permite, dar nu este corect, si trebuie sa avem grija la acest aspect